client/requirements-dev.txt

```diff
@@ -22,10 +22,10 @@ isort >= 5.10.1
 trio-typing[mypy] >= 0.7.0
 respx >= 0.19.0
 # for integration test
-torch
-# TODO: wait for tensorflow release for python3.11
-# https://github.com/tensorflow/tensorflow/issues/58032
-tensorflow; python_version <= '3.10'
+torch>=2.0.1; python_version >= '3.11'
+torch; python_version < '3.11'
```

Comment:
Do we need to specify the torch version when it maintains the suitable python version?

Reply:
Under python 3.11, swcli and unitest code does not need the specified pytorch and tensorflow versions.